restore_spaces/rs/plumbing.lua

```diff
@@ -195,19 +195,21 @@ return function(rs,hs)
         else
             target_space = space
         end
-        --target_space = tonumber(target_space)
+        if target_space then
```

Comment:
Can you add an `else` statement with
```
--mod.issueVerbose("message with the window ID...", mod.verbose)
```
to this if block to ensure the verbose mode shows `target_space` was not defined? This is an edge case and should not happen, so the message will help with future debugging.

Reply:
sounds good